homework3.php

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
```

Comment:
@Niki1993 
It will be more organised if you put homework3 in a separate folder : )